### PR TITLE
add bert infer predict script for sst-2, test=develop

### PR DESCRIPTION
**Pull request by qili93**
### PR types

New features

New features### PR changes

Others

### Description

Add predict script of paddle inference for SST-2 data.

### CODE REVIEW

File: examples/language_model/bert/README.md [hunk 1]
```diff
@@ -192,6 +192,49 @@ python -u ./predict_glue.py \
 - `batch_size` 表示每个预测批次的样本数目。
 - `max_seq_length` 表示最大句子长度，超过该长度将被截断。
 
+同时支持使用输入样例数据的方式进行预测任务，这里仅以SST-2数据为例，输出样例数据的预测结果，即正负情感：
```
Comment (wawltor) [inline examples/language_model/bert/README.md]: 正负情感有点歧义了，可以说是文本的情感分类取向，前面介绍SST-2的数据吧

File: examples/language_model/bert/predict.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
```
Comment (wawltor) [inline examples/language_model/bert/predict.py]: 2020->2021

File: examples/language_model/bert/predict.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+        self.tokenizer = tokenizer
+        self.max_seq_length = max_seq_length
+
+    @classmethod
+    def create_predictor(cls, args):
+        max_seq_length = args.max_seq_length
+        config = paddle.inference.Config(args.model_path + ".pdmodel",
+                                         args.model_path + ".pdiparams")
+        if args.device == "gpu":
+            # set GPU configs accordingly
```
Comment (wawltor) [inline examples/language_model/bert/predict.py]: set->Set 同下

File: examples/language_model/bert/README.md (hunk 1 above)
Comment (qili93) [reply examples/language_model/bert/README.md]: Done

File: examples/language_model/bert/predict.py (hunk 2 above)
Comment (qili93) [reply examples/language_model/bert/predict.py]: Done

File: examples/language_model/bert/predict.py (hunk 3 above)
Comment (qili93) [reply examples/language_model/bert/predict.py]: Done

**Review by wawltor — APPROVED** [review]
LGTM

**Pull request merged**